src/db/seeds.rb

```diff
@@ -5,3 +5,5 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+User.create(username: 'Student', email: 'student@hk.com', password: 'password')
```

Comment:
i think this should go to fixtures as seed is meant for things that are necessary for the app to work while fixtures are test data.